create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/app.py

```diff
@@ -3,16 +3,19 @@
 
 {%- if cookiecutter.use_redis == 'y' %}
 from functools import partial
+import aioredis
 {%- endif %}
 
 import aiohttp_jinja2
 {%- if cookiecutter.use_postgres == 'y' %}
 import aiopg.sa
 {%- endif %}
 from aiohttp import web
-{%- if cookiecutter.use_redis == 'y' %}
-import aioredis
+{%- if cookiecutter.use_sentry == 'y' %}
+import sentry_sdk
+from sentry_sdk.integrations.aiohttp import AioHttpIntegration
 {%- endif %}
+
```

Comment:
No need to separate different third-party libs.

Reply:
ok